docs/wiki/01 - STUDY PARAMETERS SETUP/01 - Study Variables/04 - Subprojects.md

```diff
@@ -0,0 +1,39 @@
+# Subprojects
+
+## Overview
+The **Subproject** plays an important role in determining the instrument battery that will be assigned to a candidate at a timepoint.
+
+**Subprojects** and **Cohorts** are terms used interchangeably in LORIS. **Subprojects** are defined in the `subproject` table of the database.
+
+
+## Adding Options
+
+### Front End (Recommended)
+Subprojects are defined in the Configuration module, which can be found in LORIS under the **Admin** menu tab.  Click on _To configure study subprojects click here_ link at the top of the page.
+  
+   You then click on _New SubprojectID_, fill in the fields on the right, click _save_. Refresh the page to view your new subproject. If you accidentally create an extra entry, you will have to delete it from the database manually with an sql command.  This process is similar to adding new projects.
+   
+> Note: Clicking _save_ more than once will register a duplicate subproject ID.
```

Comment:
no longer true

Reply:
just make sure to remove the line if not true anymore. :)

Reply:
done